spec/requests/api/v1/users/sessions_spec.rb

```diff
@@ -41,4 +41,37 @@
       end
     end
   end
+
+  path "/api/v1/users/sign_out" do
+    delete "Signs out a user" do
+      tags "Sessions"
+      produces "application/json"
+      parameter name: :Authorization, in: :header, type: :string, required: true
+
+      let(:casa_org) { create(:casa_org) }
+      let(:volunteer) { create(:volunteer, casa_org: casa_org) }
+      let(:api_credential) { create(:api_credential, user: volunteer) }
+      let(:refresh_token) { api_credential.return_new_refresh_token![:refresh_token] }
```

Comment:
Er, is this regenerating a refresh token a 2nd time?

Reply:
I think when you create the api credential table for the first time, the token digest fields are nil - at least in dev environment and so that is why I created the seed file to populate them.

I also need the plain text refresh token to be passed in the authorization header for the sign out.

Reply:
Sure thing, will make a note this for future issues.